CHANGELOG.md

```diff
@@ -4,7 +4,53 @@
 
 #### Changed
 
-- Internal CPU mining now injects nonce bytes at the end.
+## 1.7 (2019-03-26)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2020-04-02T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will stop working on **2020-04-30T00:00:00Z**.
+
+Breaking validation changes:
```

Comment:
Perhaps all these subheadings should be `####`? (`###` is a bit big, I find).

Reply:
@mightybyte @slpopejoy what do you think?

Reply:
nah let's just stay consistent

Reply:
In fact, please don't do any subheads, to be consistent with other changelog entries. Should just be bullet, description (PR number)